src/sharedHelpers/cvModel.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import { Platform } from "react-native";
+import RNFS from "react-native-fs";
+
+const modelFiles = {
+  IOSMODEL: "optimized_model.mlmodelc",
+  IOSTAXONOMY: "taxonomy.json",
+  ANDROIDMODEL: "optimized_model.tflite",
+  ANDROIDTAXONOMY: "taxonomy.csv"
+};
+
+export const dirModel: string = Platform.select( {
+  ios: `${RNFS.DocumentDirectoryPath}/${modelFiles.IOSMODEL}`,
+  android: `${RNFS.DocumentDirectoryPath}/${modelFiles.ANDROIDMODEL}`
+} );
+
+export const dirTaxonomy: string = Platform.select( {
+  ios: `${RNFS.DocumentDirectoryPath}/${modelFiles.IOSTAXONOMY}`,
+  android: `${RNFS.DocumentDirectoryPath}/${modelFiles.ANDROIDTAXONOMY}`
+} );
+
+const addCameraFilesAndroid = () => {
+  const copyFilesAndroid = ( source, destination ) => {
+    RNFS.copyFileAssets( source, destination )
+      .then( () => {
+        console.log( `moved file from ${source} to ${destination}` );
+      } )
+      .catch( error => {
+        console.log(
+          error,
+          `error moving file from ${source} to ${destination}`
+        );
+      } );
+  };
+
+  RNFS.readDirAssets( "camera" ).then( results => {
+    const model = modelFiles.ANDROIDMODEL;
+    const taxonomy = modelFiles.ANDROIDTAXONOMY;
+
+    const hasModel = results.find( r => r.name === model );
+
+    // Android writes over existing files
+    if ( hasModel !== undefined ) {
```

Comment:
This results in a silent failure if the model files aren't in the correct places. Instead, I think it should provide an informative an alarming error, or it should cause a fatal crash. These files are required for some core app functionality.

Reply:
I have added an error alert showing when there is no model available here. This does require to create a fake model though during e2e tests to avoid the error alert. I have added a node script to create such fake model. Without explanation in the Readme though, as its only use case would be here I think.